test/document_store_tests/test_knowledge_graph.py

```diff
@@ -7,8 +7,7 @@
 from haystack.utils import fetch_archive_from_http
 
 
-@pytest.mark.graphdb
```

Comment:
Oh, this shows in the diff? I though I introduced the graphdb marker in this PR :sweat_smile: I'll review this line to see what's up.

I had a bit of trouble with GraphDB because most of the document store tests fail with it, and I wasn't sure whether to treat it like a docstore, like a node, or what else. By the way, is GraphDB supposed to pass the docstore tests in your opinion? I thought it would

Reply:
I remember now. Initially the graphdb marker had to be explicitly added to a test; now instead it's automatically added the same way as the docstore markers are, so it's no more needed. See conftest.py:L159